Creating task with Remote resources fails

<!---
Copyright (C) 2020 Intel Corporation

SPDX-License-Identifier: MIT
-->

### My actions before raising this issue
- [x ] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x ] Searched [past issues](/issues)

Creating task with Remote resources fails with error message "Could not create the task. Open the Browser Console to get details". Error message in console "cvat-app.tsx:208 Error: Could not create the task on the server. TypeError: can only concatenate str (not "SSLCertVerificationError") to str."

### Expected Behaviour
I expect task to be created.

### Current Behaviour

Creating task fails with error message.

### Possible Solution

n/a

### Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to
reproduce this bug. Include code to reproduce, if relevant -->
1. Click Create task.
2. Enter task name.
3. Select project.
4. In "Select files" section click "Remote resources".
5. Enter URL of image, e.g. "https://www.byrdie.com/thmb/kmAG7kjHzYxFDkdltMYs8BrEY48=/950x0/filters:no_upscale():max_bytes(150000):strip_icc()/cdn.cliqueinc.com__cache__posts__274058__face-masks-for-pores-274058-1543791152268-main.700x0c-270964ab60624c5ca853057c0c151091-d3174bb99f944fc492f874393002bab7.jpg"
6. Click [Submit] button.

### Context
I just can't create a task. Uploading images one by one from my computer is not an option for me. By the way, uploading images from my computer also fails when I try to upload many images. But that's another story.

### Your Environment
<!--- Include as many relevant details about the environment you experienced
the bug in -->
Context is not important as same behaviour I see on cvat.org.

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,22 +8,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.3.0-alpha] - Unreleased
 
 ### Added
+
 - CVAT-3D: support lidar data on the server side (<https://github.com/openvinotoolkit/cvat/pull/2534>)
 
 ### Changed
+
 -
 
 ### Deprecated
+
 -
 
 ### Removed
+
 -
 
 ### Fixed
+
 - Kibana startup initialization (<https://github.com/openvinotoolkit/cvat/pull/2659>)
 - The cursor jumps to the end of the line when renaming a task (<https://github.com/openvinotoolkit/cvat/pull/2669>)
+- SSLCertVerificationError when remote source is used (<https://github.com/openvinotoolkit/cvat/pull/2683>)
 
 ### Security
+
 -
 
 ## [1.2.0] - 2020-01-08

--- a/Dockerfile
+++ b/Dockerfile
@@ -72,6 +72,7 @@ ENV DJANGO_CONFIGURATION=${DJANGO_CONFIGURATION}
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends install -yq \
         apache2 \
+        ca-certificates \
         libapache2-mod-xsendfile \
         libgomp1 \
         libgl1 \

